### PR TITLE
fix: no new appointment board will open if small preview is open

### PR DESCRIPTION
**Pull request by geeky-abhishek**

### CODE REVIEW

File: src/store/zustand/store.ts [hunk 1]
```diff
@@ -8,9 +8,11 @@ import create, { StoreApi, UseBoundStore } from 'zustand';
 export type AppState = {
 	calendarView: string;
 	date: Date;
+	isResumeOpen: boolean;
```
Comment (zovomat) [inline src/store/zustand/store.ts]: i find the name quite unrelated to its function, shouldn't it be isPreviewOpen?

**Comment by zovomat** [comment]
waiting on @nubsthead 's review

File: src/store/zustand/store.ts (hunk 1 above)
Comment (geeky-abhishek) [reply src/store/zustand/store.ts]: it should but the component is name event-resume-view so in order to not make a developer confuse I kept that name

File: src/view/calendar/custom-event-wrapper.jsx [hunk 2]
```diff
@@ -95,9 +97,9 @@ export const CustomEventWrapperStyler = styled.div`
 	}
 `;
 
-export default function CustomEventWrapper({ event, children, selected }) {
+export default function CustomEventWrapper({ event, children }) {
 	const anchorRef = useRef();
-	const [open, setOpen] = useState(selected);
+	const [open, setOpen] = useState(false);
```
Comment (nubsthead) [inline src/view/calendar/custom-event-wrapper.jsx:102]: why is this false instead of selected?

Comment (geeky-abhishek) [reply src/view/calendar/custom-event-wrapper.jsx:102]: @nubsthead I checked the props by spreading it and found out select does not exist so I remove it

Comment (nubsthead) [reply src/view/calendar/custom-event-wrapper.jsx:102]: selected should exist only when you select an event, can you retry selecting one?

Comment (geeky-abhishek) [reply src/view/calendar/custom-event-wrapper.jsx:102]: Yes did ,it's not there

**Review by nubsthead — APPROVED** [review]

**Review by dhavaldodiya — APPROVED** [review]

**Pull request closed without merging**